DDG4/src/Geant4StepHandler.cpp

```diff
@@ -125,6 +125,7 @@ double Geant4StepHandler::birkAttenuation() const    {
   const G4Track* trk = step->GetTrack();
   const G4ParticleDefinition* particle = trk->GetDefinition();
   const G4MaterialCutsCouple* couple = trk->GetMaterialCutsCouple();
+  s_emSaturation.SetVerbose(true);
```

Comment:
`SetVerbose` is still called with a boolean instead of an int.
You could "squash" the two commits into one and force push to this branch, e.g., by using `git rebase --interactive`